.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended"
```

Comment:
you can have a look at Airbnb plugin/ extension which comes with predefined linting rules.